### ISSUE TITLE
Stuck at Installation page ..

### ISSUE THREAD

**Issue by xenium888** [sdetweil/MagicMirror_scripts#29]
Just pasted the bash -c  "$(curl -sL https://raw.githubusercontent.com/sdetweil/MagicMirror_scripts/master/raspberry.sh)" code and updating packages .."am waiting for a long time for the next instruction while my raspberry pi terminal is "updating packages"
What should I do next? Newbie ...

**Comment by sdetweil** [comment]
what base is did u start with?  buster requires the fewest updates. Stretch and Jessie require more

**Comment by xenium888** [comment]
Hi. Not sure what you meant. Anyway I think I need to reinstall raspbian OS as my Pi setup lost its sound output after I rebooted. Will attempt to reinstall magic mirror again later.
Thanks for responding to my cry for help.

**Comment by sdetweil** [comment]
there are different versions of raspi

the current version code name is buster
the prior version was stretch
the prior version was jessie

some people are following an (old)  document of steps,  which causes them to download an older version of Raspi OS

can u post the  log,  (if u still have it) ~/install.log

**Comment by xenium888** [comment]
So you can see I am really a Raspberry Pi newbie. Just bought the Pi and trying to learn how to turn this into something useful. Sorry if I respond late as I reside in Singapore and we are about 12 hours apart. It's almost 10pm and I will try to work on it tomorrow and hopefully, I will have something to show for my effort after this weekend. Cheers.

**Comment by sdetweil** [comment]
no need to apologize, we are all newbies at some point..
take your time, no rush for me... i will be here..

**Comment by xenium888** [comment]
Hi Sam.
Just wanted to update you on where I am now wrt mm2.
I reinstalled my pi with the latest raspbian OS (buster?) and the first thing I did was to test if the sound output jack worked. Well it still didn’t. 
Anyways, I followed the rest of the mm2 installation steps, or should I say missteps (had to use the new password instead of the very clear instruction to use the raspberry pi default password ie raspberry, I had to ping raspberrypi instead of just ping pi, etc) but I eventually got to the part where I was greeted by mm2 opening screen. 
I will now try to explore the basic configuration by tinkering with the weather forecast location, etc before continuing the rest of the mm2 installation steps.
Thanks for your patience, Sam.
Regards,
Peter
